dtfs-central-api/src/services/repositories/utilisation-reports-repo.test.js

```diff
@@ -1,4 +1,4 @@
-const { saveUtilisationReportDetails, getUtilisationReportDetails } = require('./utilisation-reports-repo');
+const { saveUtilisationReportDetails, getUtilisationReportDetailsByBankId } = require('./utilisation-reports-repo');
```

Comment:
Shall we add a super simple test to this file testing the collection is called with the right ID?